brain/python/arrayHelpers.cpp

```diff
@@ -239,6 +239,52 @@ bp::object toNumpy( const brain::Matrix4f& matrix )
     return bp::object( bp::handle<>( array ));
 }
 
+
+bp::object frameToTuple(CompartmentReportFrame&& frame)
+{
+    return bp::make_tuple(frame.getTimestamp(), toNumpy(frame.takeData()));
+}
+
+bp::object framesToTuple(CompartmentReportFrames&& frames)
+{
+    if(frames.empty())
+        return bp::make_tuple(-1.f,std::vector<float>());
+    if(frames.size() == 1)
+        return frameToTuple(std::move(frames.front()));
```

Comment:
This is not what I understood from the spec. Indeed, I found it strange that, depending on the frames size, it changed the layout of the returned value, but : 
 def load(timestamp):
......
The return type is a (double, nadarray) tuple where the first element is the
        actual timestamp of the frame and the second one is a 1D double ndarray."""